tenets/codelingo/go/insecure-http-connection/codelingo.yaml

```diff
@@ -0,0 +1,58 @@
+tenets:
+  - name: insecure-http-connection
+    actions:
+      codelingo/review:
+        comment: Skipping verification exposes the connection to man-in-the-middle attacks and should never be used in production code.
```

Comment:
s/ and should never be used in production code./ .This should be used only for testing.